www/docs/news/editme.php

```diff
@@ -48,9 +48,9 @@
 12 => array('OpenAustralia Goes Mobile and Gets Social', <<<EOT
 <div style="float: right;padding: 5px;"><img src="/images/mobile-oa.png"></div>
 
-Many of you may have been in the same situation as this - you're out having a quiet drink with friends and a loud political discussion ensues. Without access to OpenAustralia.org to settle the debate, you're stuck. Well now that's changing - from today when you access OpenAustralia on your mobile device, you'll get a site optimised for its small display so you can quickly and easily settle that "discussion" :) Thanks to <a href="http://twitter.com/nambor">Rob Manson</a> for creating this great new feature.
+Many of you may have been in the same situation as this - you're out having a quiet drink with friends and a loud political discussion ensues. Without access to OpenAustralia.org to settle the debate, you're stuck. Well now that's changing - from today when you access OpenAustralia on your mobile device, you'll get a site optimised for its small display so you can quickly and easily settle that "discussion" :) Thanks to Rob Manson for creating this great new feature.
 
-A feature that we added to the site recently, created by yours truly, is the ability to share speeches on social networks like Facebook and Twitter. You may have noticed that there is a "Share This" link next to each speech listed on OpenAustralia. Simply hover over that link and select the social network you want to share the speech on and start a discussion with your friends.
+A feature that we added to the site recently, created by yours truly, is the ability to share speeches on social networks. You may have noticed that there is a "Share This" link next to each speech listed on OpenAustralia. Simply hover over that link and select the social network you want to share the speech on and start a discussion with your friends.
```

Comment:
Shall i leave this news intact? It's announcing a feature that we're now removing.